plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/ResultSetStatListener.java

```diff
@@ -57,25 +61,31 @@ public void handleResultSetSelectionChange(SelectionChangedEvent event) {
         @Override
         public IStatus runInUIThread(IProgressMonitor monitor) {
             IResultSetSelection selection = viewer.getSelection();
-            if (selection instanceof IResultSetSelectionExt) {
+            DBPPreferenceStore preferenceStore = viewer.getPreferenceStore();
+            if (selection.getSelectedAttributes().equals(Collections.emptyList())) {
```

Comment:
You can move this check into `setSelectionStatistics` method.

And it actually should be called `updateSelectionStatistics` instead.